pkg/release/release.go

```diff
@@ -196,3 +201,68 @@ func URLPrefixForBucket(bucket string) string {
 	}
 	return urlPrefix
 }
+
+// extractImageManifest takes an image tar file, extracts the manifest and return the tag
+func extractTagFromManifest(tarFile string) (imageManifest string, err error) {
+	if !util.Exists(tarFile) {
+		return "", errors.New("cannot extract manifest, image path does not exist")
+	}
+	// Temp type to hold the manifest tags
+	type manifestTags []struct {
+		RepoTags []string `json:"RepoTags"`
+	}
+
+	// Extract the manifest from the tar file
+	output, err := command.New("tar", "xf", tarFile, "manifest.json", "-O").RunSilentSuccessOutput()
+	if err != nil {
+		return "", errors.Wrap(err, "while extracting manifest.json from tarfile")
+	}
+
+	// Unmarshal the manifest
+	manifest := manifestTags{}
+	if err := json.Unmarshal([]byte(output.Output()), &manifest); err != nil {
+		return "", errors.Wrap(err, "unmarshalling manifest file")
+	}
+	// Is this an error?
+	if len(manifest[0].RepoTags) == 0 {
+		return "", errors.New("unable to find RepoTags in manifest")
+	}
+
+	return manifest[0].RepoTags[0], nil
+}
+
+// ReadReleaseImageTags Takes a workdir and returns the release images from the manifests
+func ReadReleaseImageTags(workDir string) (imagesList map[string][]string, err error) {
```

Comment:
```suggestion
func GetImageTags(workDir string) (imagesList map[string][]string, err error) {
```

Personal nit: for functions named `ReadFoo`, I expect some bytes to be return (reading in a file, for example). Since we're returning a map of string slices, let's use `GetFoo`

Reply:
Sounds like a good naming strategy to me. Changed.